coforma/swift-tech-challenge
# Data ingestion part 2

# Using generative AI for content creation

# Summary
This modifies the previous data ingestion flow to incorporate generative AI to create images and decsriptive text for the institutions. This is done using Lambdas that will speak to the Amazon Bedrock API Jurrassic and Titan models

## Type of change


# How To Test
1. Add a file to institutions folder in the bucket that has all the data, make it a new institution id
2. Verify institution ends up in Dynamo and that an image is generated an placed in the images bucket
_Note: This should all happen with triggers once the original file is added_

# Checklist:
